app.py

```diff
@@ -524,9 +524,9 @@ def get(self):
 
 
 endpoints = [
-    (r"/r/insights/platform/upload", RootHandler),
-    (r"/r/insights/platform/upload/api/v1/version", VersionHandler),
-    (r"/r/insights/platform/upload/api/v1/upload", UploadHandler),
+    (r"/api/ingress/", RootHandler),
```

Comment:
minor nitpick: new url has trailing /, old one does not. I don't know if this URL is actually used or not.